api/v1alpha3/condition_consts.go

```diff
@@ -120,10 +117,28 @@ const (
 )
 
 const (
-	// MachineOwnerRemediatedCondition is set on machines that have failed a healthcheck by the MachineHealthCheck controller.
+	// MachineOwnerRemediatedCondition is set on machines that have failed a healthcheck by the Machine's owner controller.
```

Comment:
Revert this? The older one was correct, MHC sets the condition

Reply:
I was trying to cover the case where we don't use MHC for KCP remediation, but currently this is not the case. Reverted.